elastic/kibana
# [Component templates] Privileges support

This PR adds functionality to verify the user has the required `manage_index_templates` cluster privileges.

### How to test

1. Created two roles: one with the `manage_index_templates` cluster privilege and one with not. Both should have access to load Kibana (e.g., assign `kibana_admin`).
2. The user without the `manage_index_templates` privilege should not be allowed to access the Component Templates tab in Index Management. The user with the cluster privilege should be able to load the UI.

### Screenshots
<img width="1304" alt="Screen Shot 2020-06-10 at 9 49 42 PM" src="https://user-images.githubusercontent.com/5226211/84339542-aae25a00-ab6c-11ea-83dd-7bc152ddf241.png">
